docs/reference/camera.md

```diff
@@ -293,6 +293,8 @@ period = wb_camera_get_sampling_period(tag)
 *enable and disable camera updates*
 
 The `wb_camera_enable` function allows the user to enable a camera.
+Once the camera is enabled, it will copy images from VRAM to RAM at each time step, regardless of `wb_camera_get_image` calls.
```

Comment:
```suggestion
Once the camera is enabled, it will copy images from GPU memory to CPU memory at each time step, regardless of `wb_camera_get_image` calls.
```